lib/virt_autotest/virtual_storage_utils.pm

```diff
@@ -90,6 +90,8 @@ sub destroy_virt_storage_pool {
 sub delete_physical_volume {
     my $lvm_disk_name = shift;
     my $timeout = 180;
+    # Enable NVME Hard Drive Support
+    $lvm_disk_name = ($lvm_disk_name =~ "nvme") ? "${lvm_disk_name}p" : $lvm_disk_name;
```

Comment:
Does this apply to all nvme cases ?

Reply:
Yes, this changed would be applied to our all testing NVME Hard Drive. But, Still need more VR to validate.